test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -417,6 +418,44 @@ public void testAclSqlResultSetHasOneRecord() throws SQLException {
     assertEquals(golden, acl);
   }
 
+  @Test
+  public void testSingleDocSqlResultSetHasOneRecord() throws SQLException {
+    executeUpdate("create table data("
+        + "id int, col1 varchar(20),"
+        + GsaSpecialColumns.GSA_PERMIT_GROUPS + " varchar(20),"
+        + GsaSpecialColumns.GSA_PERMIT_USERS + " varchar(20),"
+        + GsaSpecialColumns.GSA_DENY_GROUPS + " varchar(20),"
+        + GsaSpecialColumns.GSA_DENY_USERS + " varchar(20))");
+    executeUpdate("insert into data("
```

Comment:
This test should minimally have 2 rows as a test of STOP_NEXT.

Reply:
Thx, added another row